seed/seed.go

```diff
@@ -157,6 +157,9 @@ type Seed interface {
 	// Iter provides a way to iterately perform a function on
 	// each of the snaps for which LoadMeta loaded their metadata.
 	Iter(f func(sn *Snap) error) error
+
+	// Auto import assertion from root of seed if grade is dangerous
+	LoadAutoImportAssertion(commitTo func(*asserts.Batch) error) error
```

Comment:
nitpick: `commitTo` does not look very well, when you do `commitTo(batch)` below, it sounds like you are committing something to the batch.

A few alternatives:
- onAssertionsLoaded
- onBatchLoaded
- loadCallback

Reply:
This is actually following convention already used in here, see
[seed20.go#L84](https://github.com/snapcore/snapd/blob/master/seed/seed20.go#L84)

Reply:
I'd still vote for changing it in new code at least, but it's a nitpick anyway :-)

Reply:
@mardy then I'd propose new PR where we change all, to have it consistent and not have half one way and another half in other way...

Reply:
That would be excellent, thanks!